src/ParkingAPI.java

```diff
@@ -2,14 +2,30 @@
 
 public class ParkingAPI {
 
-    
+	public ParkingAPI() {
 
-    public ParkingAPI() {
+	}
 
-    }
+	public int getOpenSpots(ParkingLot parkingLot) {
+		Random random = new Random(parkingLot.ordinal());
+		return random.nextInt(110);
+	}
+	
+	public int getFilledSpots(ParkingLot parkingLot) {
+		Random random = new Random(parkingLot.ordinal());
+		return random.nextInt(110);
+	}
 
-    public int getOpenSpots(ParkingLot parkingLot) {
-        Random random = new Random(parkingLot.ordinal());
-        return random.nextInt(110);
-    }
-}
\ No newline at end of file
+	public ParkingLot availableParkingLots(ParkingLot one, ParkingLot two) {
+		int availableLotOne = getOpenSpots(one) - getFilledSpots(one);
+		int availableLotTwo = getOpenSpots(two) - getFilledSpots(two);
```

Comment:
This logic seems off

With this logic, say we have a parking lot with 10 spots, and 4 of them are full. `getOpenSpots()` would be returning `6`, and `getFilledSpots()` would be returning `4`. The logic here would claim that there are `2` open spots.